oap-server/server-starter/src/main/resources/otel-rules/aws-eks/eks-cluster.yaml

```diff
@@ -0,0 +1,52 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This will parse a textual representation of a duration. The formats
+# accepted are based on the ISO-8601 duration format {@code PnDTnHnMn.nS}
+# with days considered to be exactly 24 hours.
+# <p>
+# Examples:
+# <pre>
+#    "PT20.345S" -- parses as "20.345 seconds"
+#    "PT15M"     -- parses as "15 minutes" (where a minute is 60 seconds)
+#    "PT10H"     -- parses as "10 hours" (where an hour is 3600 seconds)
+#    "P2D"       -- parses as "2 days" (where a day is 24 hours or 86400 seconds)
+#    "P2DT3H4M"  -- parses as "2 days, 3 hours and 4 minutes"
+#    "P-6H3M"    -- parses as "-6 hours and +3 minutes"
+#    "-P6H3M"    -- parses as "-6 hours and -3 minutes"
+#    "-P-6H+3M"  -- parses as "+6 hours and -3 minutes"
+# </pre>
+
+filter: "{ tags -> tags.job_name == 'aws-cloud-eks-monitoring' }" # The OpenTelemetry job name
+expPrefix: tag({tags -> tags.cluster = 'aws-eks-cluster::' + tags.ClusterName})
+expSuffix: service(['cluster'], Layer.AWS_EKS)
+metricPrefix: eks_cluster
+metricsRules:
+  - name: node_count
+    exp: cluster_node_count.downsampling(LATEST)
+  - name: failed_node_count
+    exp: cluster_failed_node_count.downsampling(LATEST)
+  - name: namespace_count
+    exp: namespace_number_of_running_pods.sum(['Namespace','cluster'])
+  - name: service_count
+    exp: service_number_of_running_pods.sum(['Service','cluster'])
+  - name: net_rx_dropped
+    exp: node_network_rx_dropped.downsampling(SUM)
+  - name: net_rx_error
+    exp: node_network_rx_errors.downsampling(SUM)
+  - name: net_tx_dropped
+    exp: node_network_tx_dropped.downsampling(SUM)
+  - name: net_tx_error
+    exp: node_network_tx_errors.downsampling(SUM)
```

Comment:
Can you confirm why these metrics use `downsampling(SUM)`, I can see these units are `Count/Second` from the
origin OTEL metrics. `SUM` would plus the value of the metrics each time, should use the default  downsampling 
 `AVG`?

Reply:
please ignore the approval for now... It's a mistake.

Reply:
Sum may be not correct, especially at high dimensiona(hour/day)

Reply:
@pg-yang Could you check here? `Network TX Error Count (per second)` is the description, why it could be `downsampling(SUM)`?

Reply:
Yep, I will confirm these issues by the end of this week at the latest.

Reply:
I think, with `SUM` downsampling, it is more likely, we should not need `per second`. Because the value would be error count in minute/hour/day as a SUM.

Reply:
The unit of `node_network_tx_errors `(original data source) is `Count/Second`, so whether use `SUM` or `AVG`, the result may be incorrect.

Reply:
If we got the metrics values in 1 min for example:
6/s and 10/s
I think the AVG value 8/s is more accuracy than the SUM value 16/s

Reply:
It relies on how it is measures. Usually, a thing, if it is calculated count per second, and it is updated with time, then it means there is a window and buffer mechanism in there. We just need the LATEST downsampling policy to save and visualize data.
The key is, is this the case? If there is one error happens in T1, how this metric shows on T1, T1+1s, and T1+1s. 

From what I learned, there is nearly impossible to have count per second always, there has to be a range.

Reply:
> If we got the metrics values in 1 min for example:
> 
> 6/s and 10/s
> 
> I think the AVG value 8/s is more accuracy than the SUM value 16/s

If so, latest is the best. Because the latest status matters for APM.

Reply:
> If so, latest is the best. Because the latest status matters for APM.

Agree, I think should to confirm the time window range. If the metric value is calculated from the beginning till current.

Reply:
Yes, it is.

Reply:
I couldn't find any documentation that describes the calculation, but I found the source code.
In a word, the calculation logic is `(preValue-currentValue) / (preTimeInSecond-currentTimeInSecond)`.

1. value (`preValue` or `currentValue`) is cumulative count of transmit errors encountered. It's defined in https://github.com/google/cadvisor/blob/master/info/v1/container.go#L434
2. `currentValue`, `currentTimeInSecond` will be stored to cache as `preValue`, `preTimeInSecond` for the next calculation

I put the source code here:
1.  Calculate method: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/951245d4273845de59db3b00f5445bcad9ce4372/internal/aws/metrics/metric_calculator.go#L67
2. The `calculateFunc ` method that is called by `Calculate method`: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/951245d4273845de59db3b00f5445bcad9ce4372/receiver/awscontainerinsightreceiver/internal/cadvisor/extractors/extractor.go#L127

BTW
```
  - name: net_tx_error
    exp: node_network_tx_errors.downsampling(SUM)
```
The above metric is organized from node level to cluster level,so I think `latest ` should be inappropriate.Also,if we use `avg`, the meaning will be net errors per second, per node.

Reply:
From your description, I would say avg makes more sense.
Sum for day and hour dimension would be a disaster, right?
We have to keep per node, I am afraid, there is no way we could sum first and then avg, because from data process perspective, there is no difference between two node and two period report of one node, right?

Reply:
Agree.

Reply:
![image](https://user-images.githubusercontent.com/3917424/210130644-62919b8b-f9eb-458a-8e6f-70a623a24a79.png)